opendebates/models.py

```diff
@@ -271,6 +272,7 @@ def user_display_name(self):
 
     display_name = models.CharField(max_length=255, null=True, blank=True)
     twitter_handle = models.CharField(max_length=255, null=True, blank=True)
+    phone_number = PhoneNumberField(null=True, blank=True)
```

Comment:
We seem to repeat this pattern elsewhere, but anything that becomes a `varchar` in the DB shouldn't have `null=True` (form behavior should remain the same, but you will have to provide a default value of `''` when creating the migration).